js/private/js_library.bzl

```diff
@@ -46,45 +46,45 @@ _ATTRS = {
     "srcs": attr.label_list(
         doc = """Source files that are included in this library.
```

Comment:
nit: when I'm paranoid and de-dent docs to the file level, I start with backslash-newline so the first line is also at column 0
```
          """\
First line goes here
```